### PR TITLE
Make the repo more friendly

### PR DESCRIPTION
**Pull request by elsom25**
- [x] Evolve root README to be more approachable and clear on purpose
- [x] Adjust code structure to better isolate dev-specific concerns

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,83 @@
...
-- `parquet`
+### 📦 Integrating with the taxonomy
 
-### Source & Data files
+Dive straight into `/dist` to find the files you need and integrate this taxonomy into your system.
 
-Used to manage and update the taxonomy itself, along with necessary files to generate published formats reliably. When proposing adjustments to Shopify's product taxonomy, submit changes to these files.
+We're working on a variety of formats to make it easy to integrate with your systems. Today we have `txt` and `json` formats, and we're working on more. If you have a specific format you'd like to see, please open an issue and let us know!
 
+### 🧑🏼‍🏫 Changing the taxonomy
```
Comment (AgathaK) [inline README.md:69]: Lets add commentary that we are  making this open-source for changes (soon)

File: README.md [hunk 2]
```diff
@@ -1,77 +1,83 @@
...
+| Software | 💁🏼‍♀️ soon |
+| Cameras & optics | 💁🏼‍♀️ soon |
+| Hardware | 💁🏼‍♀️ soon |
+| Baby & toddler | 💁🏼‍♀️ soon |
+| Business & industrial | 💁🏼‍♀️ soon |
+| Office supplies | 💁🏼‍♀️ soon |
+| Services | 💁🏼‍♀️ soon |
 
-### Distribution files
+## 🗄️ How we organize things
```
Comment (AgathaK) [inline README.md:57]: Something about the sub-headers is not resonating. MaybeL

## 🗄️ How we organize things --> How to use this GitHb repo

### 📦 Integrations - How to integration with the taxonomy

### 📦 Making Changes - How to submit changes to  the taxonomy

File: README.md [hunk 3]
```diff
@@ -1,77 +1,83 @@
-# Shopify's Standard Product Taxonomy ([v0.0.1](./VERSION) - PREVIEW)
+<p align="center"><img src="./docs/assets/img/header.png" /></p>
```
[the remark this answers is not in the document]
Comment (elsom25) [reply README.md:1]: Its generated from GPT! But should be short lived (cc @sainihas)

File: README.md [hunk 4]
```diff
@@ -1,77 +1,83 @@
...
-| Hardware | Coming soon |
-| Baby & toddler | Coming soon |
-| Business & industrial | Coming soon |
-| Office supplies | Coming soon |
-| Services | Coming soon |
-
-## Organization
-
-This github repository includes both the source code for defining the taxonomy, as well as distribution files for consuming the taxonomy. We welcome public input to evolve and adjust this taxonomy. Requested changes will be reviewed and considered for subsequent version updates.
+| Apparel & Accessories | ✅ done |
```
[the remark this answers is not in the document]
Comment (Chris911) [reply README.md:36]: +1 or maybe no emoji for "Coming soon"? IMO it's a tad too overwhelming with all the emojis 

Screenshot 2024-02-01 at 9 25 44 AM

[the remark this answers is not in the document]
Comment (elsom25) [reply README.md:36]: Did a straw poll and folks like the clock emoji. Gonna keep it

File: README.md [hunk 5]
```diff
@@ -1,77 +1,87 @@
...
-- `parquet`
+Dive straight into `dist/` to find the files you need and integrate this taxonomy into your system.
 
-### Source & Data files
+We're working on a variety of formats to make it easy to integrate with your systems. Today we have `txt` and `json` formats, and we're working on more. If you have a specific format you'd like to see, please open an issue and let us know!
 
-Used to manage and update the taxonomy itself, along with necessary files to generate published formats reliably. When proposing adjustments to Shopify's product taxonomy, submit changes to these files.
+### 🧑🏼‍🏫 How to make changes to the taxonomy: `data/`
 
+{% note %}
```
[the remark this answers is not in the document]
Comment (elsom25) [reply README.md:71]: Ah rude. Must only work in "docs" and not READMEs

**Pull request merged**